wmn-data.json

```diff
@@ -5137,7 +5137,18 @@
         "m_string" : "<title>Salon24 - Blogi, wiadomości, opinie i komentarze",
         "m_code" : 301,
         "known" : ["matuzalem", "niewiarygodne"],
-        "cat" : "blog"
+        "cat" : "blog"
+       },
+       {
+        "name" : "Scammer.info",
+        "uri_check" : "https://scammer.info/u/{account}.json",
+        "uri_pretty" : "https://scammer.info/u/{account}",
+        "e_code" : 200,
+        "e_string" : "<title> Profile -",

```

Comment:
The JSON being returned is not showing this HTML tag. Please consider changing this to `\"id\":`